assets/js/applicationinsights.js

```diff
@@ -1,13 +1,26 @@
+/* global Microsoft */
```

Comment:
A special eslint comment to declare the name `Microsoft` as a global (which it is by virtue of the App Insights library inclusions on the page)
This simply means our IDEs do not moan about the name `Microsoft` not being declared in lines 8 and 16 👍